src/dialdbg/README.md

```diff
@@ -40,6 +40,11 @@ viam-dialdbg --uri myremote.com --credential mycredential --output ./foo.txt --c
 ```
 Same as above, but only examines WebRTC connection establishment.
 
+``` shell
+viam-dialdbg --uri myremote.com --credential mycredential --output ./foo.txt --credential-type api-key --entity myentity --nogrpc
```

Comment:
I realize this example is a little complex. I'll add an internal wiki page of how to use `viam-dialdbg` and how to interpret its output (I may end up copying some of that page into this `README`, too, for hypothetical external users).